### PR TITLE
Setup-phone- portfolio

### PR DESCRIPTION
**Pull request by tchasinga**
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
Linter should be set up correctly for this repository.
Linter result should be green.
There should be no custom changes to linter config.
Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
README file should be descriptive.
Commit messages should be meaningful.
Pull Request should have a descriptive title explaining what changes are inside.
Pull Request should have a short summary describing introduced changes in more detail.
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
Branch name should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
Pull Request is made from 'feature branch' into the main branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,299 @@
...
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
```
Comment (algerina) [inline README.md:40]: - [ ] Kindly replace this section with the title of your project and remove the logo.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,299 @@
...
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio: setup and mobile first <a name="about-project"></a>
+
+> A portfolio website is a digital platform where an individual or a company showcases their work or services to potential clients or employers, highlighting their skills, experience, and expertise through an organized and visually appealing layout.
+
+**SETUP PORTFOLIO ** is a...a digital platform
+
```
Comment (algerina) [inline README.md:71]: - [ ] Kindly remove this line.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,299 @@
...
+### Tech Stack <a name="tech-stack"></a>
+
+> I can provide a general overview of the tech stack typically used for a portfolio website:
+
+Front-end: HTML, CSS, JavaScript, and popular frameworks/libraries such as React, Vue, or Angular.
+Back-end: a server-side language such as Node.js, Python, or PHP, along with a web framework like Express, Flask, or Laravel.
+Database: MySQL, MongoDB, or PostgreSQL, depending on the project's needs.
+Deployment: cloud platforms like AWS, Azure, or Google Cloud, along with containerization tools such as Docker and orchestration tools like Kubernetes.
+Of course, the specific tech stack may vary depending on the project's requirements, budget, and other factors.
+
```
Comment (algerina) [inline README.md:83]: - [ ] This section is about the technology used to build your portfolio: HTML, CSS, JS.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,299 @@
...
+   > Yes, most open-source projects typically provide documentation and examples to help new developers get started with using the project. This documentation may include information on how to install and configure the project, as well as tutorials and code samples demonstrating how to use its various features and capabilities.
+
+In addition to the official documentation, many open-source projects also have active communities of developers who can provide support and guidance to new users. These communities may offer forums, chat rooms, or other channels where developers can ask questions, share ideas, and get help with any issues they encounter while using the project.
+
+If you are interested in using an open-source project, it's a good idea to start by exploring the available documentation and reaching out to the project's community for support and guidance. This can help you get up and running quickly and ensure that you are using the project effectively and efficiently.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
```
Comment (algerina) [inline README.md:291]: - [ ] Kindly update the Readme with your project information and remove any unnecessary sections.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,299 @@
...
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: My licence pages is located inside of my repository here's the links
```
Comment (algerina) [inline README.md:296]: - [ ] The link to the MIT license is broken, kindly add a license and fix this issue.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  <body>
+    <!--Here's the introduction of the part of this page
+ at this top you'll fund the presentation porfolion of developer 
+-->
+    <header id="headermain">
+        <img src="Img/Union.png" class="menupages" alt="">
+      <nav>
+        <ul>
+          <h4 class="letterttilte">Jack</h4>
+          <li class="about">Portfolio</li>
```
Comment (algerina) [inline index.html:21]: - [ ] For semantic reasons, the logo should be wrapped in an `a` tag.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    <link rel="web icon" href="Icon/resize.webp" />
+
+    <title>Setup-portfolio</title>
+  </head>
+  <body>
+    <!--Here's the introduction of the part of this page
+ at this top you'll fund the presentation porfolion of developer 
+-->
+    <header id="headermain">
+        <img src="Img/Union.png" class="menupages" alt="">
```
Comment (algerina) [inline index.html:17]: - [ ] The hamburger button should be wrapped in a `nav` tag.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+        </li>
+        <li class="links">
+             <img src="Svg/Linkedin icon.png" alt="">
+        </li>
+        <li class="links">
+            <img src="Svg/medium.png" alt="">
+        </li>
+        <li class="links">
+            <img src="Svg/angellist.png" alt="">
+        </li>
```
Comment (algerina) [inline index.html:54]: - [ ] The list of icons should use a ul tag to group them, and li per icon; each icon has a link (a tag).
Apart from being semantically correct, the tag will let you connect your social networks with the icons, using href.

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+
+  body {
+    text-decoration: none;
+    background-image: url('Icon/Header\ bg.png');
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 99vh;
+  }
+
+  header nav ul {
```
Comment (algerina) [inline styles.css:93]: - [ ] The background color is different from the one in the Figma template, kindly stick to the Figma design.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @tchasinga ,

Great job so far! 
However you still have a minor issues to fix.

May I highlight the following: 
- [x] Your project looks good👏  ✔️ 
- [x] No Linter errors✔️ 


## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by tchasinga** [comment]
Dear code reviewed I've done your all requirements thanks your help please, approved my project and let me to move forward

File: README.md [hunk 10]
```diff
@@ -1,2 +1,299 @@
...
+
+>Here you can deploy the project live by searching source course 
+Using the following website 
+<[text](https://www.w3schools.com/html/default.asp)>
+
+ it will be helpful for you  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
```
Comment (iambenkis) [inline README.md]: - [ ] As the previous said Kindly update the README with your project information and remove any unnecessary sections, like
  - how to setup your project 
  - how to run it
  - how to us it

File: README.md (hunk 5 above)
Comment (iambenkis) [inline README.md]: - [ ] As the previous reviewer said he link to the MIT license is broken, kindly add a license and fix this issue.
error

File: index.html [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+  </head>
+  <body>
+    <!--Here's the introduction of the part of this page
+ at this top you'll fund the presentation porfolion of developer 
+-->
+    <header id="headermain">
+        <img src="Img/Union.png" class="menupages" alt="">
+      <nav>
+        <ul>
+          <h4 class="letterttilte">Jack</h4>
```
Comment (iambenkis) [inline index.html]: - [ ] As the previous reviewer said, for semantic reasons, the logo should be wrapped in an `a` tag. It'll be something like `<a href="" > logo</a>`

File: index.html (hunk 8 above)
Comment (iambenkis) [inline index.html]: - [ ] As the previous reviewer said The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (a tag).
Apart from being semantically correct, the tag will let you connect your social networks with the icons, using href. It'll be something like 👍 

````
<ul>
     <li class="links">
             ...icon
     </li>
</ul>

````

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @tchasinga 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tchasinga** [comment]
Dear hello I've already done everything you ask me to now you can by your self thank you

**Comment by tchasinga** [comment]
Good checking time for you

**Review by V-Blaze — APPROVED** [review]
Hi @tchasinga  , 

# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] Project Looks like the Figma design
- [ ] Implemented all the changes from the previous reviewers

## [Optional] suggestions:
 - [OPTIONAL] Good job on customizing your readme file for your project and making it look professional, However, I noticed there were lots of grammatical errors and wrong spelling of some words, Please kindly look into this, Try to use some grammar checking app to get the best result, 😄 😄 

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**